packages/cf-util-http/src/http.js

```diff
@@ -123,14 +127,21 @@ export function request(method, url, opts, callback) {
   url = opts.url;
   callback = opts.callback;
 
+  // Emulate superagent's ability to automatically encode JSON body and set the
+  // Content-Type
+  if (BODYLESS_METHODS.every(m => method.toUpperCase() !== m) && opts.body) {
+    opts.body = JSON.stringify(opts.body);
+    opts.headers = new Headers(opts.headers || {});
+    opts.headers.set('Content-Type', 'application/json');
```

Comment:
Does this replace the `Content-Type` header if it already exists or add a second one?  I can't find docs for `.set()` they all link to `Set()`(Data structure)